saagieapi/projects/projects.py

```diff
@@ -53,8 +54,7 @@ def list(self) -> Dict:
         dict
             Dict of projects information
         """
-        query = gql(GQL_LIST_PROJECTS)
-        return self.saagie_api.client.execute(query)
+        return self.saagie_api.client.execute(query=gql(GQL_LIST_PROJECTS), pprint_result=True)
```

Comment:
`pprint_result` as parameter?

Reply:
not needed as this function is not called by other functions